lib/mini_profiler/timer_struct/request.rb

```diff
@@ -5,15 +5,15 @@ class MiniProfiler
     module TimerStruct
       class Request < TimerStruct::Base
 
-        def self.createRoot(name, page)
-          TimerStruct::Request.new(name, page, nil).tap do |timer|
+        def self.createRoot(name, more_path, page)
```

Comment:
I am so sorry I am pushing this back and forth so often but I do feel we are mixing concerns here.

From the profilers perspective this is simply a concern of `TimerStruct::Page` no? 

It gets the environment, and it can then look up:

- What is the path of this request?
- What is the verb of the request? 

It can then pass those two fields back to the client which in turn can decide what do do (how to create the more path and what to put in it)